WooCommerce/Classes/ViewRelated/Orders/ScanToPay/ScanToPayView.swift

```diff
@@ -0,0 +1,74 @@
+import SwiftUI
+import WooFoundation
```

Comment:
```suggestion
```
Seems that this import is not needed! We only use `FullScreenCoverClearBackgroundView` on the `PaymentMethodsView`.

Reply:
Good catch! Fixed in 5ae5989634f7dfc021508083d1a65a07f39ccde7